targets/nvidia-jetson-orin/default.nix

```diff
@@ -49,6 +49,11 @@
           ../../modules/virtualization/microvm/microvm-host.nix
           ../../modules/virtualization/microvm/netvm.nix
           ({config, ...}: {
+            services.udev.extraRules = ''
+              # Make microvm the owner of all vfio devices
+              SUBSYSTEM=="vfio",OWNER="microvm",GROUP="kvm"
```

Comment:
Could we perhaps get little less broad rule, maybe only for the UART device itself?

Reply:
Fixed with permissions to the IOMMU group